compiler/rustc_ast/src/ast.rs

```diff
@@ -2092,6 +2092,10 @@ pub enum TyKind {
     Never,
     /// A tuple (`(A, B, C, D,...)`).
     Tup(ThinVec<P<Ty>>),
+    /// An anonymous struct type i.e. `struct { foo: Type }`
+    AnonymousStruct(ThinVec<FieldDef>),
+    /// An anonymous union type i.e. `union { bar: Type }`
+    AnonymousUnion(ThinVec<FieldDef>),
```

Comment:
Just sharing an observation that I feel like i typically see the shorthand `Anon` in the AST and HIR more often than the complete `Anonymous`. Not sure if there's any standard practice/desire for consistency though :shrug:

Reply:
Oh, yes, I agree with you :smile: 
```
grep -i \\banon\[^y\] -R compiler | wc -l
520
grep -i \\banonymous -R compiler | wc -l 
277
grep -i \\banon\[^y\] -R compiler/rustc_ast* compiler/rustc_hir* | wc -l
184
grep -i \\banonymous -R compiler/rustc_ast* compiler/rustc_hir* | wc -l
46
```